tokens/f555c46bad0731d080c9381d7fff6f82839946a66bd070d185e1ea2f.BlackDogeCoin.json

```
@@ -0,0 +1,9 @@
+{
+    "policyId": "f555c46bad0731d080c9381d7fff6f82839946a66bd070d185e1ea2f",
+    "name": "BlackDogeCoin",
```

```suggestion
    "name": "BDOGE",
```

This needs to be your asset name. If I see correctly, that is also "BDOGE" (https://cardanoscan.io/token/f555c46bad0731d080c9381d7fff6f82839946a66bd070d185e1ea2f42444f4745)